go/vt/vtconsensus/config/vtconsensus_config.go

```diff
@@ -0,0 +1,573 @@
+/*
+Copyright 2021 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+
+	"gopkg.in/gcfg.v1"
+
+	"vitess.io/vitess/go/vt/log"
+)
+
+// VTConsensusConfig is the config for VTConsensus
+type VTConsensusConfig struct {
```

Comment:
lacks unittest

Reply:
I didn’t write vtconsensus UT, I’ll make up for it in the past few days